tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -91,6 +93,9 @@
 autograph = lazy_loader.LazyLoader(
     "autograph", globals(),
     "tensorflow.python.autograph.impl.api")
+interleave_ops = lazy_loader.LazyLoader(
+    "interleave_ops", globals(),
+    "tensorflow.python.data.experimental.ops.interleave_ops")
```

Comment:
Add a comment for why we need to lazy load interleave_ops.

Reply:
addressed in the commit